libraries/ms-common/src/stm32f0xx/interrupt_mcu.c

```diff
@@ -0,0 +1,92 @@
+#include "interrupt_mcu.h"
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "interrupt.h"
+#include "status.h"
+#include "stm32f0xx_exti.h"
+#include "stm32f0xx_misc.h"
+#include "stm32f0xx_rcc.h"
+#include "stm32f0xx_syscfg.h"
+
+// Channels are NVIC channels which handle prioritization of interupts.
+#define NUM_INTERRUPT_MCU_CHANNELS 32
```

Comment:
It might be nice to mention where you got this information from just so we have some context :)